src/components/WalletUnlock/DeterministicWallet.tsx

```diff
@@ -247,35 +265,66 @@ const DeterministicWallet = ({
           <Trans id="MNEMONIC_TITLE" />
         </Title>
       </HeadingWrapper>
-      <Typography>
-        <Trans id="MNEMONIC_SUBTITLE" />
-      </Typography>
-      <Parameters>
-        <AssetSelector
-          selectedAsset={assetToUse}
-          showAssetIcon={false}
-          showAssetName={true}
-          searchable={true}
-          assets={filteredAssets}
-          onSelect={(option: ExtendedAsset) => {
-            handleAssetUpdate(option);
-          }}
+
+      <Box pb={SPACING.MD} variant="spaceBetween">
+        <SBox variant="columnAlignLeft">
+          <Text>
+            <Trans id="MNEMONIC_SUBTITLE" />
+          </Text>
+          <AssetSelector
+            selectedAsset={assetToUse}
+            showAssetIcon={false}
+            showAssetName={true}
+            searchable={true}
+            assets={filteredAssets}
+            onSelect={(option: ExtendedAsset) => {
+              handleAssetUpdate(option);
+            }}
+          />
+        </SBox>
+        <SBox
+          variant="columnAlignLeft"
+          style={{ visibility: !displayEmptyAddresses ? 'hidden' : 'visible' }}
+        >
+          <Text>
+            <Trans id="MNEMONIC_DPATH_SELECT" />{' '}
+            <LinkApp href="" onClick={() => setDpathAddView(true)}>
+              <Trans id="DETERMINISTIC_CUSTOM_LINK_TEXT" />
+            </LinkApp>
+            {'.'}
+          </Text>
+          <Selector
+            value={selectedDPath}
+            onChange={setSelectedDPath}
+            options={dpaths}
+            optionComponent={DPathOption}
+            valueComponent={({ value }) => <DPathOption data={value} />}
+            clearable={false}
+            searchable={false}
+          />
+        </SBox>
+      </Box>
+      <Box variant="rowAlign" justifyContent="space-between" pb={SPACING.SM}>
+        <Switch
+          checked={displayEmptyAddresses}
+          onChange={() => toggleDisplayEmptyAddresses(!displayEmptyAddresses)}
```

Comment:
This isn't a toggle when you have to define the param. Maybe rename to `setDisplayEmptyAddresses`